### PR TITLE
Use scorecard v1alpha3

### PR DESCRIPTION
**Pull request by fabianvf**
**Description of the change:**
We will now use the newly defined Scorecard v1alpha3 API instead of v1alpha2

**Motivation for the change:**
v1alpha2 has been superseded by v1alpha3

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -103,6 +103,7 @@ images/scorecard-test/scorecard-test
 pkg/ansible/runner/testdata/valid.yaml
 /bin
 test/test-framework/go.sum
+internal/scorecard/alpha/examples/custom-scorecard-tests/go.sum
 
 # Website
 website/public/
```
Comment (camilamacedo86) [inline .gitignore:109]: should we not add a fragment for that?

Comment (fabianvf) [reply .gitignore:109]: I don't think so, shouldn't have any impact on the user. Fragment probably required for some other stuff htough

Comment (camilamacedo86) [reply .gitignore:109]: is not it changing the json output? see [here](https://github.com/operator-framework/operator-sdk/pull/3208/files#diff-b86df888a60472475e24887d92f4c4aaL162-R176)

Comment (fabianvf) [reply .gitignore:109]: It looks like that link just takes me back to this conversation

**Pull request merged**